src/app/stacktracedialog.h

```diff
@@ -61,21 +61,18 @@ class StacktraceDialog : public QDialog, private Ui::StacktraceDialog
             "</p></font>"
             "<br/><hr><br/>"
             "<p align=center><font size=4>"
-#if defined(__x86_64__) || defined(_M_X64)
-            "qBittorrent version: " QBT_VERSION " (64-bit)<br/>"
-#else
-            "qBittorrent version: " QBT_VERSION " (32-bit)<br/>"
-#endif
-            "Libtorrent version: %1<br/>"
+            "qBittorrent version: " QBT_VERSION " (%1-bit)<br/>"
```

Comment:
You can leave existing order of arguments to have less unnecessarily changed lines. Just use `%7` here.

Reply:
I know, but I still prefer having it in order, a bit easier for me to read, besides the changes are already made, lets just keep it.